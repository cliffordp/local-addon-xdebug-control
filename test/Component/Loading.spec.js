const React = require( 'react' )
const Component = require( 'react' ).Component
const renderer = require( 'react-test-renderer' )

const context = {
	'React': React,
}

const Loading = require( './../../src/Component/Loading' )( context )

describe( '<Loading/>', function () {
	it( 'renders correctly with props', function () {
		const component = renderer.create( <Loading/> )
		let tree = component.toJSON()

		expect( tree ).toMatchSnapshot()
	} )
} )
