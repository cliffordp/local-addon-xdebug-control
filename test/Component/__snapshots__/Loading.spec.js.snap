// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> renders correctly with props 1`] = `
<div
  className="BigLoader"
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
    }
  }
>
  <div
    className="LoadingIndicator LoadingIndicator--Big"
  >
    <div
      className="LoadingIndicator_Bounce1"
    />
    <div
      className="LoadingIndicator_Bounce2"
    />
    <div
      className="LoadingIndicator_Bounce3"
    />
  </div>
</div>
`;
